suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -61,6 +65,25 @@ export const XpubScanScreen = ({
     useFocusEffect(resetToDefaultValues);
 
     const goToAccountImportScreen = ({ xpubAddress }: XpubFormValues) => {
+        const isCoinWithXpub =
```

Comment:
This would be better to determine based on network type, not the network symbol. Then with just one equal comparison, you can handle the whole expression.
```suggestion
        import { networks } from '@suite-common/wallet-config';
        const networkType = networks[networkSymbol];
        const isCoinWithXpub = networkType === "bitcoin";
```

Creating an helper function in suite-common `hasXpub(networkSymbol) => networkType` would be even better.

Reply:
https://github.com/trezor/trezor-suite/pull/8893/commits/45dc3f0adcdab5146b85621af61fa8b577c21d01